config/submitter.go

```diff
@@ -11,6 +11,9 @@ const (
 	DefaultPollingIntervalSeconds    = 60   // in seconds
 	DefaultResendIntervalSeconds     = 1800 // 30 minutes
 	DefaultResubmitFeeMultiplier     = 1
+	DefaultInsufficientFeeMargin     = 0.15
```

Comment:
can we add some rationale why `0.15` is good default and how operators should chose those config values 
```
	InsufficientFeeMargin     float64 `mapstructure:"insufficient_fee_margin"`     // e.g. 0.15 for 15%
	InsufficientFeerateMargin float64 `mapstructure:"insufficient_feerate_margin"` // e.g. 0.15 for 15%
	FeeIncrementMargin        float64 `mapstructure:"fee_increment_margin"
```

i.e I am thinking of the case when somebody from devops team will try to change those value, then he should have good idea what control what